### DISCUSSION TITLE
/How does one add line counters to raw (code) blocks

### DISCUSSION THREAD

**Discussion by Blakeinstein** [typst/typst#1989] [Q&A]
I dont see an implementation of any kind of line counter, is it not a feature yet?

I am currently styling my code blocks as follows

```
show raw: el => {
    block(
      stroke: black,
      fill: luma(240),
      inset: 8pt,
      radius: 4pt,
      el
    )
  }
```

**Comment by Enivex** [comment]
There are a couple of packages that can do this: algo and codelst
